app/controllers/logs_controller.rb

```diff
@@ -271,19 +271,23 @@ def update
   # can be given a single id or a list of ids
   def take
```

Comment:
:information_source: It'd be nice to get some specs around this. The `LogsController` controller has too many responsibilities. Would be cool to extract an object for `#take` which creates the new volunteers. This would allow us to unit test it better. Then we could make use of that object within `#take` only need to test the response. Food for thought.